fineract-provider/dependencies.gradle

```diff
@@ -119,7 +119,7 @@ dependencies {
         exclude group: 'commons-logging'
     }
 
-    implementation 'io.swagger.core.v3:swagger-jaxrs2-jakarta:2.2.19'
```

Comment:
Wouldnt it be better to move versions into the `org.apache.fineract.dependencies.gradle` file? I was wondering whether it would be better to have only 1 configuration file that contains the versions.

Reply:
Done, readded